app/assets/stylesheets/_global.scss

```diff
@@ -0,0 +1,18 @@
+html {
+  box-sizing: border-box;
+  font-size: 10px;
+}
+
+body {
+  color: $color-text;
+  font-family: sans-serif;
+  font-size: 1.6rem;
+  line-height: 1.6;
+  margin: 0;
+}
+
+footer,
+header {
+  display: block;
+}
```

Comment:
I pulled applicable css from normalize.css and put it in the global scss file, thinking that was a good place for it but in this case it makes sense that it be included with the relevant partial.